### PR TITLE
Option to not show progressbar.

### PR DESCRIPTION
**Pull request by michaelkaye**
This was on matrix-org-hotfixes ; is required for running the script in a regular systemd timer rather than on the commandline.

### CODE REVIEW

**Comment by michaelkaye** [comment]
(I feel i could have written this better, 6 months ago)

File: scripts/s3_media_upload [hunk 1]
```diff
@@ -406,6 +420,8 @@ def main():
     )
 
     args = parser.parse_args()
+    if args.no_progress:
+        progress=False
```
Comment (erikjohnston) [inline scripts/s3_media_upload:424]: Don't you need a `global progress`?

Comment (michaelkaye) [reply scripts/s3_media_upload:424]: Apparently only if you want the ability to not specify `--no-progress` is this important; looks like the current matrix-org-hotfixes just doesn't support progress bars at all.

File: scripts/s3_media_upload [hunk 2]
```diff
@@ -26,6 +27,7 @@ SCHEMA = """
     CREATE INDEX IF NOT EXISTS deleted_idx ON media(known_deleted);
 """
 
+progress=True
```
Comment (clokep) [inline scripts/s3_media_upload:30]:
```suggestion
progress = True
```

File: scripts/s3_media_upload [hunk 3]
```diff
@@ -406,6 +420,9 @@ def main():
     )
 
     args = parser.parse_args()
+    if args.no_progress:
+        global progress
+        progress=False
```
Comment (clokep) [inline scripts/s3_media_upload:425]:
```suggestion
        progress = False
```

**Review by clokep — COMMENTED** [review]
A couple of minor formatting things.

**Pull request merged**